### ISSUE TITLE
Сбивающее с толку сообщение об отсутствии БД при добавлении ИБ в кластер без флага "Создавать БД в случае отсутствия" 

### ISSUE THREAD

**Issue by kuntashov** [YanSergey/OneS_ClusterAdmin#21]
Добавляю новую ИБ в кластер копированием.

В окне со "скопированными" свойствами меняю имя ИБ в кластере и имя СУБД, указываю, как в примере на скриншоте `etalon_ut`.

Флажок "Создавать БД в случае отсутствия" не устанавливаю (не намеренно - забыл, но не суть).

Указываю пароль СУБД.

Нажимаю "ОК" и получаю ошибку "Сервер баз данных не обнаружен" (скриншот), что удивляет: СУБД точно на месте. Потом вчитываюсь внимательно и вижу, что реальная ошибка "база данных etalon_ut не существует".

![2022-12-29_00-08](https://user-images.githubusercontent.com/1042299/209867383-835ec47f-89ba-49c7-a545-2667556d9224.png)

Ожидаю, что в окне с ошибкой будет выводиться только информация об отсутствии указанной базы данных в СУБД.

**Comment by YanSergey** [comment]
Да, вижу такое. Что интересно - это сам сервер 1С возвращает такую ошибку.
Видимо, придется искусственно вырезать лишний текст.

**Comment by YanSergey** [comment]
Выяснил, что при выполнении этого действия через штатную консоль ошибка точно такая же.
Получается что это сам Постгрес сбивает с толку некорректным сообщением.
Не вижу смысла с этим что-то делать.
![2024-01-09_11-11-49](https://github.com/YanSergey/OneS_ClusterAdmin/assets/24665801/9aae0651-ffcc-4120-866d-4ac713146a2a)
